[nltk_data] Downloading package punkt to /home/dtujo/nltk_data...
[nltk_data]   Package punkt is already up-to-date!
[nltk_data] Downloading package averaged_perceptron_tagger to
[nltk_data]     /home/dtujo/nltk_data...
[nltk_data]   Package averaged_perceptron_tagger is already up-to-
[nltk_data]       date!
2021-06-13 16:16:27.172851
06/13/2021
06-13-2021
Traceback (most recent call last):
  File "main.py", line 18, in <module>
    grabFinance.grabFinance()
  File "/mnt/optane/dtujo/Trawler/scrap/Trawler_v2/grabFinance.py", line 30, in grabFinance
    dateTuple = records[0]
IndexError: list index out of range
